src/panels/logbook/ha-logbook.ts

```diff
@@ -288,13 +289,13 @@ export class HaLogbook extends LitElement {
       return;
     }
     try {
-      this._subscribed = await subscribeLogbook(
+      const subscribePromise = subscribeLogbook(
         this.hass,
         (streamMessage) => {
           // "recent" means start time is a sliding window
           // so we need to calculate an expireTime to
           // purge old events
-          if (!this._subscribed) {
+          if (!subscribePromise) {
```

Comment:
This will never evaluate to `true`. It looks like it was put there to handle the case where the component is unloading but events keep coming in. I don't think it did it properly though as it would only work after `unsubscribe` and at that point there shouldn't be any more events coming.
In any case, we should figure out another condition here or remove it if it is not needed.